shared/js/toc.jsx

```diff
@@ -312,13 +310,13 @@ loop.shared.toc = (function(mozL10n) {
           <div className="tile-info">
```

Comment:
With a `userName` instead of `userId` we can just directly use the name for the `room-user` strings above.